src/components/Wallet.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import * as TezosRPCTypes from "conseiljs/dist/types/tezos/TezosRPCResponseTypes";
 import {TezosNodeReader} from "conseiljs";
-import {useContext, useEffect, useState} from "react";
+import {useCallback, useContext, useEffect, useState} from "react";
```

Comment:
Since the ticket asks you to just change the Settings component, you should be making any changes to the Wallet component unless you are making some global changes which affect it.